Wikipedia/Code/NSDateFormatter+WMFExtensions.h

```diff
@@ -39,6 +39,6 @@
 
 + (instancetype)wmf_mediumDateFormatterWithoutTime;
 
-+ (instancetype)wmf_hyphenatedYearMonthDayFormatter;
++ (instancetype)wmf_englishHyphenatedYearMonthDayFormatter;
```

Comment:
@bgerstle i was looking up the locale stuff as well - it seems to have the most effect on the preset styles (long medium short) - as it formats them specifically with each locale. 

With custom formats, that all it does is basically what you said: Just change characters/numbers. (Also it seems the formatter can fail because not all things work in all locales)

Makes me wonder if for non-english languages if we should use the built in styles because they are formatted in a way the user is guaranteed to understand. Rather than forcing an english convention.